ui/lib/core/addon/components/choose-pgp-key-form.hbs

```diff
@@ -30,18 +30,17 @@
         @container="#shamir-flow-modal"
       />
     </div>
-    <div class="field is-grouped">
-      <div class="control">
-        <button type="button" class="button" {{on "click" (fn (mut this.selectedPgp) "")}} data-test-confirm-pgp-back-button>
-          Back
-        </button>
-      </div>
-      <div class="control">
-        <button type="submit" disabled={{not this.pgpKey}} class="button is-primary" data-test-confirm-pgp-key-submit>
-          {{this.buttonText}}
-        </button>
-      </div>
-    </div>
+    <Hds::ButtonSet>
```

Comment:
followed HDS guidelines as much as possible given the constraints of this nested component and made this a tertiary button 
<img width="754" alt="Screenshot 2023-11-13 at 4 36 07 PM" src="https://github.com/hashicorp/vault/assets/68122737/eba8aa89-5a11-4f2c-8a4c-431879a85f62">

Reply:
(the cancel button is rendered by a parent component so it can't render on the same line as the HDS docs suggest)